newrelic/structures_newrelic_workload.go

```diff
@@ -0,0 +1,163 @@
+package newrelic
+
+import (
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+	"github.com/newrelic/newrelic-client-go/pkg/nerdgraph"
+	"github.com/newrelic/newrelic-client-go/pkg/workloads"
+)
+
```

Comment:
Are the methods in this file going to be used in other resource? If not, I'd argue keeping all these contained in the same file just for maintainability?

Reply:
This is following the existing patterns we have in place elsewhere.  We'll consider a cross-cutting refactoring to simplify, but not as part of this PR.